tools/go-cli/go-whisk-cli/commands/namespace.go

```diff
@@ -79,6 +79,9 @@ var namespaceGetCmd = &cobra.Command{
         if err != nil {
             whisk.Debug(whisk.DbgError, "client.Namespaces.Get(%s) error: %s\n", qName.namespace, err)
             errStr := fmt.Sprintf("Unable to obtain namespace entities for namespace '%s': %s", qName.namespace, err)
+            if(client.Config.BaseURL.Host == "") {
```

Comment:
@lzbj, generating an error here will only produce an error for the namespace command. I suggest moving the error to the setupClientConfig() method in commands.go.